packages/fxa-support-panel/src/app/app.controller.spec.ts

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+
+import { RemoteLookupService } from '../remote-lookup/remote-lookup.service';
+import {
+  formattedSigninLocations,
+  formattedSubscriptions,
+  MOCKDATA,
+} from '../remote-lookup/remote-lookup.service.spec';
+import { AppController } from './app.controller';
+
+describe('AppController', () => {
+  let controller: AppController;
+  let mockRemote: any;
+  let mockLog: any;
```

Comment:
Making it all optional didn't seem better since casts to `any` would still be needed to pass it everywhere. Using an `any` at least reduces the casts elsewhere.

Reply:
While that's true in other files I tried this locally and tests passed.